File: src/main/java/com/github/sorusclient/client/adapter/IPlayerInventory.java
package com.github.sorusclient.client.adapter;

public interface IPlayerInventory {

    Slot getSelectedSlot();

    IItem getItem(Slot slot);

    enum Slot {
        ONE,
        TWO,
        THREE,
        FOUR,
        FIVE,
        SIX,
        SEVEN,
        EIGHT,
        NINE
    }

}

File: src/main/kotlin/com/github/sorusclient/client/adapter/IPlayerInventory.kt
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.adapter

interface IPlayerInventory {
    val selectedSlot: Slot
    fun getItem(slot: Slot): IItem?
    enum class Slot {
        ONE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE
    }
}